packages/server/migration/1699381717454-fix-organization.ts

```diff
@@ -1,47 +0,0 @@
-import { MigrationInterface, QueryRunner } from 'typeorm';
-
-export class fixOrganization1699381717454 implements MigrationInterface {
-  name = 'fixOrganization1699381717454';
-
-  public async up(queryRunner: QueryRunner): Promise<void> {
-    await queryRunner.query(
-      `CREATE TABLE "chatbot_interactions_model" ("id" SERIAL NOT NULL, "timestamp" TIMESTAMP NOT NULL, "course" integer, "user" integer, CONSTRAINT "PK_dfa22bd83de6c88afe6caa5ae37" PRIMARY KEY ("id"))`,
```

Comment:
I don't think we should delete this?

Reply:
Me neither but i could not run migrations without deleting it.